src/Form/ReactionRuleEditForm.php

```diff
@@ -80,6 +86,19 @@ public function save(array $form, FormStateInterface $form_state) {
   }
 
   /**
+   * Form submission handler for the 'cancel' action.
+   *
+   * @param array $form
```

Comment:
On form submission handlers we can skip the docs for $form and $form_state, see https://www.drupal.org/coding-standards/docs#forms